samwell/sam/__init__.py

```diff
@@ -604,3 +604,9 @@ def set_pair_info(r1: AlignedSegment, r2: AlignedSegment, proper_pair: bool = Tr
     insert_size = isize(r1, r2)
     r1.template_length = insert_size
     r2.template_length = - insert_size
+
+
+@enum.unique
+class SamOrder(enum.Enum):
+    Coordinate = "Coordinate"
+    QueryName = "QueryName"
```

Comment:
1. missing docs
2. the value can just be the value we'd expect in the sam header, so you can use it later
3. good to have unsorted to use later (could use unknown too)
```suggestion
@enum.unique
class SamOrder(enum.Enum):
    """Enumeration of major sort orders in a SAM/BAM"""
    Unsorted = "unsorted  #: the sort is unsorted 
    Coordinate = "coordinate"  #: coordinate sorted
    QueryName = "queryname"  #: queryname sorted
```